ChangeLog.md

```diff
@@ -14,6 +14,10 @@
 
 - Adds a `NonDetC` carrier for `NonDet` in `Control.Effect.NonDet.Maybe`. ([#227](https://github.com/fused-effects/fused-effects/pull/227))
 
+- Defines `Carrier` instances for a number of types in `base`, including `Either`, `Maybe`, `[]`, and `IO`. ([#206](https://github.com/fused-effects/fused-effects/pull/206))
+
+- Defines `Carrier` instances for a number of types in `transformers`. ([#226](https://github.com/fused-effects/fused-effects/pull/226))
```

Comment:
We should probably add, like, a `docs/carriers.md` document detailing what the deal is with what `Carrier` people should use.